src/components/PostComment.jsx

```diff
@@ -1,35 +1,28 @@
 import { postComment } from "../../axios"
-import { useEffect, useState } from "react"
+import { useState } from "react"
 
-export default function PostComment ({article_id, setAuthor}) {
+export default function PostComment ({article_id, setPostedComment}) {
 
-    const [comment, setComment] = useState({})
+    // const [req, setReq] = useState({})
     const [isPosted, setIsPosted] = useState(false)
     const [isLoading, setIsLoading] = useState(false)
     const [isErr, setIsErr] = useState(false)
     const [inputVal, setInputVal] = useState("")
 
     const handleInput = (e) => {
         e.preventDefault()
-
-        const obj = {
-            username: "grumpy19",
-            body: e.target.value
-        }
-        setComment(obj)
         setInputVal(e.target.value)
     }
 
     const handleSubmit = (e) => {
         e.preventDefault()
         setIsLoading(true)
         setIsPosted(true)
-        setAuthor("grumpy19")
         setInputVal("")
-        postComment(article_id, comment)
+        postComment(article_id, { username: "grumpy19", body: e.target.value })
```

Comment:
I can post empty comments, should there be something stopping this from happening ?